students/zelichowski_michal/lesson_02_flow_control/the_number_of_elements_equal_to_the_maximum.py

```diff
@@ -0,0 +1,15 @@
+"""A sequence consists of integer numbers and ends with the number 0.
+Determine how many elements of this sequence are equal to its largest
+element. """
+
+a = int(input())
+max = 0
```

Comment:
W pythonie w odróżnieniu od np. Javy funkcje są obiektami i możesz je dowolnie przypisywać do zmiennych. Oznacza to, że możesz sobie coś nadpisać: 
```
>>> max
<built-in function max>
>>> moj_max = max <- przypisałem funkcję max do moj_max
>>> moj_max
<built-in function max>
>>> max = 0 <- przypisałem 0 do max
>>> max
0
```
Jak masz wątpliwości czy nazwa jest wolna to możesz sprawdzić w konsoli pythona:
```
>>> max                                       
<built-in function max>                       
>>> moja_zmienna                              
Traceback (most recent call last):            
  File "<stdin>", line 1, in <module>         
NameError: name 'moja_zmienna' is not defined 
>>>                                           
```

Zalecam poprawkę, żeby przypadkowo nie przesłaniać built-inów